pull

```diff
@@ -86,241 +52,147 @@ while [ "$1" != "" ]; do
+        cd $serverPath/ && mkdir -p tmp
+        docker cp main-application-1:/var/www/html/content/ tmp/
+    "
+    catch "Could not extract texts."
 
-# Skip database synchronization, if remote database user is empty
-if [[ -z ${remoteDatabaseUser} ]];
-then
-  echo 'Skip database synchronization'
-  echo '-----------------------------'
-else
-  # Export database
-  echo 'Export database'
-  echo '---------------'
-  if [ $hoster = 'mittwald' ];
-  then
-    serverPath='/home/www/'$remoteServerUser
-  else
-    serverPath='/home/forge/'$remoteServerHost
-  fi
-  ssh -T $remoteServerUser@$remoteServerHost -p $remoteServerPort "
-    cd $serverPath/ && mkdir -p tmp
-    mysqldump --ignore-table=$remoteDatabaseName.be_sessions --ignore-table=$remoteDatabaseName.cache_md5params --ignore-table=$remoteDatabaseName.cache_treelist --ignore-table=$remoteDatabaseName.cache_hash --ignore-table=$remoteDatabaseName.cache_hash_tags --ignore-table=$remoteDatabaseName.cache_imagesizes --ignore-table=$remoteDatabaseName.cache_imagesizes_tags --ignore-table=$remoteDatabaseName.cache_pages --ignore-table=$remoteDatabaseName.cache_pages_tags --ignore-table=$remoteDatabaseName.cache_pagesection --ignore-table=$remoteDatabaseName.cache_pagesection_tags --ignore-table=$remoteDatabaseName.cache_rootline --ignore-table=$remoteDatabaseName.cache_rootline_tags --ignore-table=$remoteDatabaseName.cache_treelist --ignore-table=$remoteDatabaseName.cf_extbase_datamapfactory_datamap --ignore-table=$remoteDatabaseName.cf_extbase_datamapfactory_datamap_tags --ignore-table=$remoteDatabaseName.cf_extbase_object --ignore-table=$remoteDatabaseName.cf_extbase_object_tags --ignore-table=$remoteDatabaseName.cf_extbase_reflection --ignore-table=$remoteDatabaseName.cf_extbase_reflection_tags --ignore-table=$remoteDatabaseName.tx_extensionmanager_domain_model_extension --ignore-table=$remoteDatabaseName.sys_log --user="$remoteDatabaseUser" --password='$remoteDatabasePassword' --host="$remoteDatabaseHost" $remoteDatabaseName > tmp/$databaseFilename
-  "
-  scp -P $remoteServerPort $remoteServerUser@$remoteServerHost:$serverPath'/tmp/'$databaseFilename ./
-  echo ''
+    # Download texts
+    rsync -chavzP -e "ssh -p $remoteServerPort" --delete --stats $serverAddressAndPath/tmp/content/ content/
+    catch "Could not download texts."
+    color green "Successfully downloaded texts."
+fi
 
-  # Import database
-  echo 'Import database'
-  echo '---------------'
-  if [ $environment = 'docker' ];
-  then
-    docker cp $databaseFilename $dockerDatabaseHost:/var/www/html/$databaseFilename
-    docker-compose exec -T database mysql --user="$dockerDatabaseUser" --password="$dockerDatabasePassword" $dockerDatabaseName < $databaseFilename
-  fi
-  if [ $environment = 'local' ];
-  then
-    mysql --user="$localDatabaseUser" --password="$localDatabasePassword" --host="$localDatabaseHost" $localDatabaseName < $databaseFilename
-  fi
-  echo ''
+# Export database
+headline '⬆️ Export database'
```

Comment:
I get the following error during the export routine, but the export seems to work just fine:

```
Error: 'Access denied; you need (at least one of) the PROCESS privilege(s) for this operation' when trying to dump tablespaces
```

Reply:
This is actually a big problem, because the default user does not have the needed rights to dump the database. I wrote about this in Slack back in 2023. Until now I used TablePlus to export the database as a workaround.

Reply:
Shouldn't we be able to use `--no-tablespaces` to prevent this error message?